frontend/test/__support__/e2e/cypress-plugins.js

```diff
@@ -41,7 +36,8 @@ module.exports = (on, config) => {
    **                          WEBPACK                               **
    ********************************************************************/
 
-  on("file:preprocessor", webpack(webpackPluginOptions));
+  // on("file:preprocessor", webpack(webpackPluginOptions));
+  on("file:preprocessor", createBundler());
```

Comment:
@akiselev this is a default bundle without any options. It's possible to include custom options (see [this](https://github.com/bahmutov/cypress-esbuild-preprocessor#esbuild-options)). If you have any ideas or suggestions, please let me know.